DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/ViewDelegate/ViewDelegateViewModel.cs

```diff
@@ -21,8 +21,8 @@ IEnumerable<DelegateCourseDetails> delegateCourses
             DelegateCourses = delegateCourses.Select(x => new DelegateCourseInfoViewModel(x)).ToList();
             Tags = FilterableTagHelper.GetCurrentTagsForDelegateUser(delegateUser);
             ProfessionalRegistrationNumber = DisplayStringHelper.GetPrnDisplayString(
-                delegateUser.HasBeenPromptedForPrn,
-                delegateUser.ProfessionalRegistrationNumber
+                DelegateInfo.HasBeenPromptedForPrn,
+                DelegateInfo.ProfessionalRegistrationNumber
             );

```

Comment:
Oh yeah, I hadn't done the method call on DelegateInfoViewModel but since two view models inherit from it, I've moved the DisplayStringHelper method call there

Reply:
That's not quite how I imagined the change. Why do we need the PRN as a property on these view models when the DelegateInfoViewModel it contains has it? Just get it off there in the view.